### DISCUSSION TITLE
Change default admin access from "pwn3d" to "admin"

### DISCUSSION THREAD

**Discussion by Marshall-Hallenbeck** [Pennyw0rth/NetExec#362] [Ideas]
I understand back in the day saying stuff was "pwn3d" was what we did (we all did it, you're lying if you didnt), but at this point many people use this tool with default settings and we should probably display something more descriptive by default.

Yes, there is the option to change what this says, but lets be honest, a lot of people don't check configuration options. I think that the tool is in a mature enough state where we can change this to be descriptive like "Admin" by default.

@NeffIsBack @mpgn @zblurx Please add your opinions.

**Comment by mpgn** [comment]
Bug "pwn3d" doesn't mean "admin" so it's not that simple

![image](https://github.com/Pennyw0rth/NetExec/assets/5891788/ce38ef70-d2f8-4155-9c7f-32c0f087226b)


https://www.netexec.wiki/getting-started/using-credentials

**Reply by Marshall-Hallenbeck** [reply]
Good point, maybe we should set a default for each protocol?

**Comment by NeffIsBack** [comment]
Naaa honestly I love the little "Pwn3d" flag having a bit of fun in here. Also this has been part of the tool forever, people are used to it and it is kinda part of the identity imo, if that makes sense.

If people don't like it they should at least take a look at the wiki how to change it with the config file. Also remember, on the first run there are like 10 statements "copying x to missing config". If you miss that you must be literally blind

**Reply by NeffIsBack** [reply]
So I really like that flag, would be sad to change it after that long imo

**Comment by Marshall-Hallenbeck** [comment]
Alright - it's decided, we're not changing it :D
